src/framework/test_filter/index.ts

```diff
@@ -0,0 +1,20 @@
+import { TestSpecID, TestCaseID } from '../id.js';
+import { TestFileLoader, TestSpecOrReadme } from '../loader.js';
+
+export * from './load_filter.js';
+
+// A pending loaded spec (.spec.ts) file, plus identifying information.
+export interface TestFilterResult {
+  readonly id: TestSpecID;
+  readonly spec: TestSpecOrReadme;
+}
+
+export interface TestFilter {
```

Comment:
This is an implementation detail of the `test_filter` module, maybe it could be hidden and only `loadFilter` and `TestFilterResult` exposed?